ci-operator/step-registry/cucushift/e2e/cucushift-e2e-commands.sh

```diff
@@ -73,9 +73,11 @@ function filter_test_by_platform() {
         case "$platform" in
             none)
                 export E2E_RUN_TAGS="@baremetal-upi and ${E2E_RUN_TAGS}"
+                oc get infrastructure cluster -o yaml | yq '.status.platform'
```

Comment:
Why not output platform before if block? I don't think it can take more effort but it can reduces amount of code.

Reply:
Several known platforms does not need this info.
And in the long term, only the `*` one will need this line of code.

Reply:
Yes, then if there is only one place output platform, it is more maintainable.

Reply:
Moving the code before if block, means all platforms will output this info, which is unexpected, because for those known platforms, I don't want this info.

Reply:
OK, I don't have more comments now, let's wait others to review.

Reply:
Alright, this is another way to make our script simple.